src/layouts/SiteDashboard/SiteDashboard.tsx

```diff
@@ -101,6 +102,14 @@ export const SiteDashboard = (): JSX.Element => {
     isLoading: isBrokenLinksLoading,
   } = useGetBrokenLinks(siteName)
 
+  const isBrokenLinksReporterEnabled = useFeatureIsOn(
+    "is_broken_links_report_enabled"
+  )
+
+  const growthbook = useGrowthBook()
+  console.log({ growthbook })
+
+  console.log({ isBrokenLinksReporterEnabled })
```

Comment:
Could we remove the log statements?

Reply:
[d0a6ce8](https://github.com/isomerpages/isomercms-frontend/commit/d0a6ce81b407058a3819534b2f8c733631b4821c)